### PR TITLE
feat: [R-106449] add assignment 9

### PR DESCRIPTION
**Pull request by Subhajit-QI**

### CODE REVIEW

File: Assignment9.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Rectangle
```
Comment (Ramyani-QI) [inline Assignment9.rb:2]: @Subhajit-QI question is missing.

File: Assignment9.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  end
+  
+  def perimeter
+    2 * (@length + @width)
+  end
+end
+  
+rectangle = Rectangle.new(5, 3)
+puts "Area: #{rectangle.area}"
+puts "Perimeter: #{rectangle.perimeter}"
```
Comment (Ramyani-QI) [inline Assignment9.rb]: @Subhajit-QI there should be a line break at the end of file.

**Comment by Ramyani-QI** [comment]
@Subhajit-QI  PR title format should be like `feat: [R-<redmine ticket no.>] add assignment 1`

File: Assignment9.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Write a class (with constructor) to define instance methods for getting area & perimeter of a rectangle.
+class Rectangle
+  def initialize(length, width)
+    @length = length
+    @width = width
+  end
+  
+  def area
+    @length * @width
```
Comment (Ramyani-QI) [inline Assignment9.rb]: can you use `attr_accessor`

Comment (Subhajit-QI) [reply Assignment9.rb:9]: instead of initialize?

Comment (Subhajit-QI) [reply Assignment9.rb]: > can you use `attr_accessor`
instead of initialize?

Comment (Ramyani-QI) [reply Assignment9.rb]: @Subhajit-QI please read about getters and setters methods and how to use that https://www.geeksforgeeks.org/ruby-getters-and-setters-method/

File: Assignment9.rb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  end
+  
+  def perimeter
+    2 * (@length + @width)
+  end
```
Comment (Ramyani-QI) [inline Assignment9.rb:16]: please use getter & setter methods in the main code

Comment (Ramyani-QI) [inline Assignment9.rb]: please use getter & setter methods in the main code

File: Assignment9.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  
+  def perimeter
+    2 * (length + width)
+  end
+end
+  
+rectangle = Rectangle.new
+
+rectangle.length = 8
+rectangle.width = 4
```
Comment (Ramyani-QI) [inline Assignment9.rb]: can you move this inside initialize method

File: Assignment9.rb [hunk 6]
```diff
@@ -0,0 +1,23 @@
+# Write a class (with constructor) to define instance methods for getting area & perimeter of a rectangle.
+class Rectangle
+  attr_accessor :length, :width
+
+  def initialize(length, width)
+    @length = length
+    @width = width
+  end
```
Comment (Ramyani-QI) [inline Assignment9.rb:8]: you can use setter method as well like this
```
def initialize(length, width)
  self.length = length
  self.width = width
end
```

**Review by Ramyani-QI — APPROVED** [review]

**Pull request merged**